browser-extension/mv3/src/service-worker/services/sessionRecording.ts

```diff
@@ -70,7 +69,7 @@ export const getTabSession = (tabId: number, callback: () => void) => {
 };
 
 export const watchRecording = (tabId: number) => {
-  chrome.tabs.create({ url: `${config.WEB_URL}/sessions/draft/${tabId}` });
+  chrome.tabs.sendMessage(tabId, { action: CLIENT_MESSAGES.VIEW_RECORDING, tabId });
```

Comment:
No need to send the message to background and then back to content script. This can be directly handled in the content script instead of sending `EXTENSION_MESSAGES.WATCH_RECORDING` event to background